### ISSUE TITLE
regression using msvc2015: compilation fails due to "_Pragma" symbol

### ISSUE THREAD

**Issue by Saadharta** [oatpp/oatpp#728]
Good afternoon oatpp maintainers; 

I am currently using the **msvc2015** compiler for a 32bit target. 
I am using the following command to build my Visual Studio Solution:
```
cmake --build build -S. -G "Visual Studio 14 2015" -A "Win32"
```

Oatpp version 1.3.0 works fine for me. (`git checkout tags/1.3.0`)

However i noticed that on current master, [4177fcc](https://github.com/oatpp/oatpp/commit/4177fcca7e24191012eabf77fe17ba835f8582ce) ,both `src/oatpp/web/server/handler/ErrorHandler.cpp:36` and `src/oatpp/core/data/stream/FileStream.cpp:26` use the `_Pragma` instead of the `#pragma` syntax, leading to the following compilation error (and build failure):
`error C2448: '_Pragma' : function-style initializer appears to be a function definition.`

I also had to add
```
#if defined(WIN32) || defined(_WIN32)
    #pragma warning(disable : 4996)
#endif
```

at `src/oatpp/web/server/handler/ErrorHandler.hpp:28` for my main projet to compile without the following error (leading to build failure) when linking the generated oatpp.lib : 

```
error C4996: 'oatpp::web::server::handler::ErrorHandler::handleError': was declared deprecated
```

Kind Regards,

Saadharta

**Comment by kushnertodd** [comment]
Saadharta,

I'm guilty. I'll fix it. Just leave the warning if there's no way to stop
it?

Todd

**Comment by kushnertodd** [comment]
I agree, I'll give generating the options a try. An item is that I have VS
2022, I can't test with your version. I can check things in on a branch and
send you a note to try it. I also want to try to install the pdb file
automatically, VS 2022 seems to want it.
